builders/buildPage.ts

```diff
@@ -3,22 +3,35 @@ import Page from "../models/Page";
 import buildPosts from "../builders/buildPosts";
 
 const buildPage = ($: CheerioSelector) => {
+  const avatar = buildAvatar($);
   const name = buildName($);
   const posts = buildPosts($);
 
-  return new Page({ name, posts });
-}
+  return new Page({ name, posts, avatar });
+};
 
 export default buildPage;
 
 const buildName = ($: CheerioSelector) => {
-  const title = $('title').text();
+  const title = $("title").text();
   let name = title;
 
-  const lastDashIndex = title.lastIndexOf('-');
+  const lastDashIndex = title.lastIndexOf("-");
   if (lastDashIndex !== -1) {
     name = title.substring(0, lastDashIndex - 1);
   }
 
   return name;
-}
+};
+
+const buildAvatar = ($: CheerioSelector) => {
+  const ogi = $('meta[property="og:image"]');
+  let avatar = ogi && ogi.attr("content");
+
+  if (!avatar) {
+    const ai = $('a[rel="theater"] img');
+    avatar = ai && ai.first().attr("src");
```

Comment:
`avatar = ai.attr("src")` should be fine. It works like jQuery, should read first element `attr`

Reply:
or even `avatar = $('a[rel="theater"] img').attr("src")`?

Reply:
I will not throw?

Reply:
Should not. I'll check it.